test/sagemaker_tests/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import sagemaker
+
+def invoke_pytorch_helper_function(ecr_image, n_virginia_ecr_image, sagemaker_session, n_virginia_sagemaker_session, multi_region_support, helper_function, helper_function_args):
```

Comment:
nit: Is this not applicable to all sagemaker tests? If it isn't, should it be located in `sagemaker_tests/pytorch/__init__.py`?

Reply:
It was initially intended to use for other framework test as well but it seems like it is not needed. Will move to the pytorch folder